src/index.js

```diff
@@ -96,13 +96,20 @@ const getCheckedRadio = function (nodes, form) {
   }
 };
 
+const escapeCSS =
+  typeof CSS === 'undefined' || typeof CSS.escape !== 'function'
```

Comment:
To make sure this doesn't trip-up any folks doing server-side rendering, is it better to test `typeof window !== 'undefined' && (typeof window.CSS === 'undefined' || typeof window.CSS.escape !== 'function')`?

I realize anything declared on `window` is technically a global and doesn't need `window.` to access it, but I don't have that much experience with server-side rendering to know all the ins and outs, but enough to have been tripped-up myself with missing `typeof window !== 'undefined'` in a few places before to be weary about it...

Reply:
I added a check for `typeof window`, but don't really have any SSR experience myself.  I'll do some research and see if there's a better way to handle it.